.markdownlint.json

```diff
@@ -1,5 +1,9 @@
 {
     "default": true,
-    "MD013": false, // 80 char line length
+    // line length
+    "MD013": {
+        "code_block_line_length": 256,
+        "heading_line_length": 96
```

Comment:
This was necessary for https://github.com/dotnet/source-build/blob/main/Documentation/planning/multi-sdk-band-support.md#a-distro-maintainer-wishing-to-build-the-initial-release-n00-of-a-nxx-band-sdk